src/Microsoft.ML.Legacy/Models/OnnxConverter.cs

```diff
@@ -25,38 +25,42 @@ public sealed partial class OnnxConverter
-        /// For example com.microsoft.cognitiveservices. This is a required field.
-        /// <see cref="InputsToDrop"/> is a string array of input column names to omit from the input mapping.
+        /// to write the ONNX representation to.</para>
+        /// <para><c><see cref="Domain"/></c> indicates the domain name of the model. ONNX uses reverse domain name space indicators.
+        /// For example com.microsoft.cognitiveservices. This is a required field.</para>
+        /// <para><c><see cref="InputsToDrop"/></c> is a string array of input column names to omit from the input mapping.
         /// A common scenario might be to drop the label column, for instance, since it may not be practically
-        /// useful for the pipeline. Note that any columns depending on these naturally cannot be saved.
-        /// <see cref="OutputsToDrop"/> is similar, except for the output schema. Note that the pipeline handler
+        /// useful for the pipeline. Note that any columns depending on these naturally cannot be saved.</para>
+        /// <para><c><see cref="OutputsToDrop"/></c> is similar, except for the output schema. Note that the pipeline handler
         /// is currently not intelligent enough to drop intermediate calculations that produce this value: this will
-        /// merely omit that value from the actual output.
+        /// merely omit that value from the actual output.</para>
         ///
-        /// Transforms that can be exported to ONNX
-        /// 1. Concat
-        /// 2. KeyToVector
-        /// 3. NAReplace
-        /// 4. Normalize
-        /// 5. Term
-        /// 6. Categorical
+        /// <para>Transforms that can be exported to ONNX</para>
+        /// <list type="number">
+        /// <item><description>Concat</description></item>
+        /// <item><description>KeyToVector</description></item>
+        /// <item><description>NAReplace</description></item>
+        /// <item><description>Normalize</description></item>
+        /// <item><description>Term</description></item>
+        /// <item><description>Categorical</description></item>
+        /// </list>
         ///
-        /// Learners that can be exported to ONNX
-        /// 1. FastTree
-        /// 2. LightGBM
-        /// 3. Logistic Regression
+        /// <para>Learners that can be exported to ONNX</para>
+        /// <list type="number">
+        /// <item><description>FastTree</description></item>
+        /// <item><description>LightGBM</description></item>
+        /// <item><description>Logistic Regression</description></item>
+        /// </list>
         ///
-        /// See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs"/>
-        /// for an example on how to train a model and then convert that model to ONNX.
+        /// <para>See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs"/>OnnxTests.cs</a>
```

Comment:
>/ [](start = 122, length = 1)

This one is extra, please remove it.

Reply:
Thanks for pointing this out. I just fixed it.